addons/im_livechat/static/src/embed/common/chat_window_patch.js

```diff
@@ -27,6 +27,9 @@ patch(ChatWindow.prototype, {
             this.props.chatWindow.show({ notifyState: this.thread?.state !== "open" });
         } else {
             await super.close();
+            if (this.livechatService.state === SESSION_STATE.CREATED) {
```

Comment:
Thank you for the update!

@tsm-odoo is that the best way to check for a thread being temporary?
Can't we use thread specific stuff (id -1?) instead?

The `CREATED` name isn't very clear to me.

Reply:
I think the preferred way would be `thread.isTransient`.